featuretools/entityset/entity.py

```diff
@@ -589,7 +592,7 @@ def _create_index(index, make_index, df):
                            "integer column", index)
         # Case 5: make_index with no errors or warnings
         # (Case 4 also uses this code path)
-        if isinstance(df, dd.DataFrame):
+        if isinstance(df, dd.DataFrame) or isinstance(df, ks.DataFrame):
```

Comment:
We should use another strategy here, see: https://github.com/FeatureLabs/featuretools/issues/887#issuecomment-648754374